circle.yml

```diff
@@ -0,0 +1,7 @@
+machine:
+  pre:
```

Comment:
Oh nice, I just found some example code using that and figured it was necessary. Thanks!